.circleci/make.sh

```diff
@@ -2,20 +2,38 @@
 TOTALP1=$(find ../examples -maxdepth 1 -type d | wc -l)
 TOTAL=$((TOTALP1 - 1))
 
+
 PRE=config.pre.yml
 MID=config.mid.yml
 END=config.end.yml
 IEND=config.iend.yml
+DEND=config.dend.yml
 cat >"${MID}" </dev/null
 cat >"${END}" </dev/null
 cat >"${IEND}" </dev/null
+cat >"${DEND}" </dev/null
 
 cat >>"${IEND}" <<END
     - "build-sink":
+        filters:
+          tags:
+            only: /[0-9]*\.[0-9]*\.[0-9]*/
         requires:
           - "hs-test"
 END
 
+cat >>"${DEND}" <<END
+    - "hold":
```

Comment:
My intention was that you would put this directly into `config.pre.yml`  and it would just require `"build-sink"`